lib/faraday/multipart/middleware.rb

```diff
@@ -1,14 +1,13 @@
 # frozen_string_literal: true
 
-require 'securerandom'
+require "securerandom"
 
 module Faraday
   module Multipart
     # Middleware for supporting multi-part requests.
-    class Middleware < Faraday::Request::UrlEncoded
-      DEFAULT_BOUNDARY_PREFIX = '-----------RubyMultipartPost'
-
-      self.mime_type = 'multipart/form-data'
```

Comment:
This needed to be changed from a class variable to an instance variable otherwise it was causing race conditions in the RSpec tests (and presumably likely in people's code as well).